dbms/src/main/java/org/polypheny/db/transaction/StatementImpl.java

```diff
@@ -173,7 +173,7 @@ public void close() {
             dataContext.getParameterValues().clear();
         }
         fileInputHandles.forEach( FileInputHandle::close );
-        dataContext = null;
+        // dataContext = null; todo dl check how we want to handle this long term
```

Comment:
ToDo?

Reply:
Removed. Handling should hold up to our current capabilities and use-cases.